Formula/n/netatalk.rb

```diff
@@ -56,6 +55,7 @@ class Netatalk < Formula
   conflicts_with "ad", because: "both install `ad` binaries"
 
   def install
+    inreplace "meson.build", "if init_cmd != ''", "if init_cmd != '' and get_option('with-init-hooks') == true"
```

Comment:
This allows .service files to be included, will look into getting it merged for the next release.